criu/pie/parasite.c

```diff
@@ -335,6 +335,19 @@ static int fill_fds_opts(struct parasite_drain_fd *fds, struct fd_opts *opts)
 
 		p->flags = (char)flags;
 
+		/*
+		 * For O_PATH opened files there is no owner at all.
+		 */
+		flags = sys_fcntl(fd, F_GETFL, 0);
+		if (flags < 0) {
+			pr_err("fcntl(%d, F_GETFL) -> %d\n", fd, flags);
+			return -1;
+		}
+		if (flags & O_PATH) {
+			p->fown.pid = 0;
+			continue;
```

Comment:
I don't like this "continue". It might be dangerous if someone will add something useful after filling up p->fown. I think we need to add a new helper which will contain the code to fill up p->fown.